internal/daemon/daemon.go

```diff
@@ -77,7 +77,7 @@ func NewDaemon(ctx context.Context, project string) *Daemon {
 }
 
 // Init initializes the Daemon with the given configuration, and starts the database.
-func (d *Daemon) Init(listenPort string, stateDir string, socketGroup string, extendedEndpoints []rest.Endpoint, schemaExtensions map[int]schema.Update, hooks *config.Hooks) error {
+func (d *Daemon) Init(listenPort string, stateDir string, socketGroup string, extendedEndpoints []rest.Endpoint, schemaExtensions []schema.Update, hooks *config.Hooks) error {
```

Comment:
Should the doc block explain how schemaExtensions should be ordered to help the caller?

Reply:
I'm not following, do you mean we should add a comment explaining that external schema updates should be passed in as a list in the order the updates should be applied?

We should have such a comment but I think it belongs in `app.Start` where a project importing microcluster would see it.